schemas/snuba-metrics.v1.schema.json

```diff
@@ -66,6 +69,7 @@
         "retention_days",
         "tags",
         "timestamp",
+        "sentry_received_timestamp",
```

Comment:
This can't be required right now, otherwise the previous message format (which is still being produced today) will fail the validation. You can make it required at a later stage once those no longer exist. Would you mind leaving one of the examples unchanged and adding a new example with the `sentry_received_timestamp`?

Reply:
oh right oops